packages/sdk-browser/src/storage.ts

```diff
@@ -40,7 +41,6 @@ export const setStorageValue = async <K extends StorageKeys>(
   const storageLocation: StorageLocation = STORAGE_VALUE_LOCATIONS[storageKey];
   const browserStorageLocation: Storage = STORAGE_LOCATIONS[storageLocation];
   browserStorageLocation.setItem(storageKey, JSON.stringify(storageValue));
-  // eventEmitter.emit('storageChange', { storageKey, storageValue });
```

Comment:
Assuming we don't need this at all at this point

Reply:
yeah can be deleted, this is for a flow where the sdk can natively watch for changes to storage values and update the react context but we're not currently doing that so it's ok to delete